ecocode-rules-specifications/src/main/rules/EC12/EC12.json

```diff
@@ -18,8 +12,14 @@
     "performance"
   ],
   "defaultSeverity": "Major",
+  "code": {
+    "impacts": {
+      "RELIABILITY": "MEDIUM"
+    },
+    "attribute": "EFFICIENT"
+  },
   "compatibleLanguages": [
     "JAVASCRIPT",
     "TYPESCRIPT"
   ]
-}
+}
```

Comment:
There must be an empty line at the end of the file

Reply:
Done. I added the spotless plugin to check newlines for all json files.